fits2caom2/src/main/java/ca/nrc/cadc/fits2caom2/Ingest.java

```diff
@@ -409,6 +409,9 @@ protected void populateObservation(Observation observation, List<IngestableFile>
                 if (a.equals(artifact))
                 {
                     artifact = a;
+                    // assign non-identifying constructor args
+                    artifact.setProductType(ptype);
+                    artifact.setReleaseType(rtype);
```

Comment:
yeah, but since this whole module is deprecated I went for the minimal change :)